eisbuk-admin/src/pages/CustomerAreaPage/CustomerAreaCalendar.tsx

```diff
@@ -74,8 +75,9 @@ const CustomerAreaCalendar: React.FC<Props> = ({
     }),
   ]);
 
-  const allSlotsByDay = useSelector(getAllSlotsByDay);
+  const memSelector = useMemo(() => getSlotsByView(view), [view]);
 
+  const allSlotsByDay = useSelector(memSelector);
```

Comment:
I used `memSelector` for the example, but semantically, here would be better to use `slotsSelector`